.rubocop_todo.yml

```diff
@@ -1,12 +1,17 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2023-09-11 17:19:21 UTC using RuboCop version 1.56.3.
+# on 2024-04-15 18:28:32 UTC using RuboCop version 1.63.1.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
 # Offense count: 2
+# Configuration parameters: AllowedMethods, AllowedPatterns, CountRepeatedAttributes.
+Metrics/AbcSize:
+  Max: 21
```

Comment:
Can you add an exception just where you need it rather than making this exemption globally?

Reply:
ok